manifest.json

```diff
@@ -1,9 +1,10 @@
 {
-    "name": "JotunnLib",
-    "version_number": "0.1.4",
+    "name": "Jotunn, the Valheim Library",
+    "version_number": "1.0.0",
     "website_url": "https://github.com/jotunnlib/jotunnlib",
-    "description": "JotunnLib is a modding library for Valheim, with the goal of making the lives of mod developers easier. It's a centralized API to allow mod developers to easily add custom items, recipes, prefabs, and more to the game, working for both servers and clients.",
+    "description": "Jötunn, the Valheim Library was created with the intent to facilitate developer creativity, unify the communities problem solving efforts, and enhance developer productivity by curating a library of common helper utilities, as well as interfaces and abstractions which aid with mod interoperability and networked gameplay consistency.",
     "dependencies": [
-        "denikson-BepInExPack_Valheim-5.4.800"
+        "denikson-BepInExPack_Valheim-5.4.901",
+		"ValheimModding-HookGenPatcher-0.0.2"
```

Comment:
is it a dep to the bepinexpack?

Reply:
Yes. This enables the modmanagers to acquire dependencies automatically

Reply:
really? you dont say :D but seriously, is hookgenpatcher mandatory for bepinexpack? cause we ship our own if the user enables it...

Reply:
What? I don't understand what you are asking. Hookgenpatcher is mandatory for us. We use it. We also use bepinex.

Reply:
bepinex is an external dep for jötunn, right. but hookgen isnt. the nuget has its own hookgen which can be activated with doprebuild. or is nexus plain dll download only?

Reply:
no, dont even have an account there.

Reply:
Yes, but players dont use nugets. players use dll's. You are looking at this with the wrong perspective.